DXRando/DeusEx/Classes/DXRandoCrowdControlTimer.uc

```diff
@@ -112,7 +112,7 @@ state Activated
 
 defaultproperties
 {
-     bCollideActors = False
+     bCollideActors=False
```

Comment:
Whitespace here causes compiler warnings because defaultproperties parsing is not smart